apis/github

```diff
@@ -1,14 +1,14 @@
 local JSON = dofile("apis/dkjson")
 
--- Build a github API url, trying to authenticate (not tested)
+local function generateAuthHeaders(auth)
```

Comment:
Ha ha,  my bad I lost one of your comments o  the other PR so I thought this was the same thing. If your good with this let me know and I'll revise.